pinot-transport/src/test/java/com/linkedin/pinot/transport/netty/NettyCloseChannelTest.java

```diff
@@ -174,7 +176,7 @@ protected void initChannel(SocketChannel ch) throws Exception {
       super.initChannel(ch);
       ch.pipeline().removeLast();
       ch.pipeline().addLast("request_handler", new MyNettyChannelInboundHandler());
-      LOGGER.info("Server Channel pipeline setup modified. Pipeline:" + ch.pipeline().names());
+      LOGGER.debug("Server Channel pipeline setup modified. Pipeline:" + ch.pipeline().names());
```

Comment:
Fixed this one but these are existing statements in test code. Not doing full cleanup in this PR